tests/dummy/app/templates/partials/buttons/stateful.hbs

```diff
@@ -0,0 +1,23 @@
+<section class='row'>
+  <h3>Stateful Buttons</h3>
+
+  {{#destructive-button destructiveAction='deleteStuff' record=model
+                        ongoingMessage='Cancel'
+                        successMessage='Deleted!'
+                        failureMessage='Failed!'}}
+    {{fa-icon 'trash'}} &nbsp; Destructive Button
+  {{/destructive-button}}
+
+  {{#destructive-button destructiveAction='deleteStuffWithError' record='hey'
+                        ongoingMessage='Cancel'
+                        successMessage='Deleted!'
+                        failureMessage='Failed!'
+                        class='margin-left-xx-sm'}}
+    {{fa-icon 'trash'}} &nbsp; Failing Button
+  {{/destructive-button}}
+
+  {{#loading-button 'hello' 'hey' slowAction='runSlowAction'
```

Comment:
Actually i wanted to ask you about it. The problem with "named" parameters is that you need to know them all. Until now, passing `record` only works, but what if we want to do something that does not take a record but something else (or more than one parameters) ?

With the positional params, we can pass as many parameters to the action we want as it will be an array (to be deconstructed by the calling application)